Formula/httpie.rb

```diff
@@ -80,7 +80,18 @@ class Httpie < Formula
   end
 
   def install
-    virtualenv_install_with_resources
+    venv = virtualenv_create(libexec, Formula["python@3.10"].opt_bin/"python3")
+    venv.pip_install resources
+
+    # We use a special file called __build_channel__.py to denote which source
+    # was used to install httpie.
+    File.write("httpie/internal/__build_channel__.py", "BUILD_CHANNEL = \"homebrew\"")
```

Comment:
I'd use `tap.user` here:

```suggestion
    File.write("httpie/internal/__build_channel__.py", "BUILD_CHANNEL = \"#{tap.user}\"")
```

It will use a GitHub username (`Homebrew` for this repo) and makes it distinguishable for forks and custom taps.

Reply:
I don't think it is really important for us, since we only need to know the build type.